storage/innobase/log/log0recv.cc

```diff
@@ -1747,8 +1747,30 @@ static byte *recv_parse_or_apply_log_rec_body(
             /* For cloned db header page has the encryption information. */
             !recv_sys->is_cloned_db) {
           ut_ad(LSN_MAX != start_lsn);
-          if (fil_tablespace_redo_encryption(ptr, end_ptr, space_id,
-                                             start_lsn) == nullptr)
+
+          byte *ptr_copy = ptr;
+          ptr_copy += 2;  // skip offset
+          ulint len = mach_read_from_2(ptr_copy);
+          ptr_copy += 2;
+          if (end_ptr < ptr_copy + len) return nullptr;
+
+          /* With Percona Server, Tables created with
+                        ENCRYPTION='N' have crypt_data
+             (CRYPT_SCHEME_UNENCRYPTED) in Page 0 */
```

Comment:
possible to align the comment? or is it clang-format messing this up?

Reply:
it is clang-format